pageserver/src/metrics.rs

```diff
@@ -186,6 +186,16 @@ static PERSISTENT_BYTES_WRITTEN: Lazy<IntCounterVec> = Lazy::new(|| {
     .expect("failed to define a metric")
 });
 
+pub(crate) static EVICTION_ITERATION_DURATION: Lazy<HistogramVec> = Lazy::new(|| {
+    register_histogram_vec!(
+        "pageserver_eviction_iteration_duration_seconds_global",
+        "Time spent on a single eviction iteration",
+        &["period_secs", "threshold_secs"],
+        STORAGE_OP_BUCKETS.into(), // TODO: could use better buckets?
```

Comment:
All of last week fits in 16.7min with (on the latest 6h with 16.7min entries):

- <10ms: 23k
- <100ms: 16k
- <1s: 17k
- <10s: 53
- <100s: 26
- <1000s: 3

I guess we could fit however multimodal distribution there. But trying to guess better buckets, I don't know how much wiser could we become. Capturing non-bucketed data would be much better, but I don't think prometheus fits into that picture.

So suggesting:

```suggest
        STORAGE_OP_BUCKETS.into(),
```

I don't know why can't I suggest.

Reply:
Not sure how they relate to this one.
I guess gc and compaction are somewhat comparable in the sense that they're also background jobs and also do some getpage requests

Reply:
Added that source for the values were for logical size calculation. Cached values do not count in these values.

Reply:
Ok, then I guess these are good enough for starters.